pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -40,8 +40,8 @@
     "System.Threading.Thread": "4.0.0",
     "System.Threading.ThreadPool": "4.0.10",
     "Libuv": "1.9.1",
-    "Microsoft.NETCore.Platforms": "1.0.2-servicing-24703-01",
```

Comment:
Please undo these as they will be updated by https://github.com/dotnet/core-setup/pull/616. You should rebase once that PR is merged.